go.mod

```diff
@@ -62,5 +62,6 @@ require (
 replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+	github.com/herumi/bls-eth-go-binary => github.com/herumi/bls-eth-go-binary v0.0.0-20220509081320-2d8ab06de53c
```

Comment:
How about using `line/ostracon`-v1.0.6 instead of using `replace`?

* https://github.com/line/ostracon/tree/v1.0.6

Reply:
> How about using line/ostracon-v1.0.6 instead of using replace?

I think so, too.

Reply:
Thank you. I'll change `line/ostracon` version from `v1.0.6-0.20220614053335-f9d9fa2cc779` to `v1.0.6`. However this change would not have been necessary in the original. :)